models/pydantic/models.py

```diff
@@ -6,12 +6,22 @@ class AnimalCreate(BaseModel):
     animal_type: str
     name: str
     birth_date: date
+    animal_breed: str
+    photo: str
 
 
 class AnimalResponse(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
     id: int
     animal_type: str
+    animal_breed: str
     name: str
     birth_date: date
+    photo: str
+
+    @property
+    def age(self) -> int:
```

Comment:
You can use computed field
https://docs.pydantic.dev/2.0/usage/computed_fields/
and in this case you'll not need to add `animal_json['age'] = animal.age`